packages/app/src/specs/SpecsList.vue

```diff
@@ -4,7 +4,14 @@
       v-model="search"
       class="pb-32px"
       :result-count="specs.length"
-      @showCreateSpecModal="emit('showCreateSpecModal')"
+      @show-create-spec-modal="emit('showCreateSpecModal')"
+      @show-spec-pattern-modal="showSpecPatternModal = true"
+    />
+    <SpecPatternModal
+      v-if="props.gql.currentProject"
+      :show="showSpecPatternModal"
+      :gql="props.gql.currentProject"
+      @close="showSpecPatternModal = false"
     />
```

Comment:
This looks like this should be added to the top component. 
Then the trigger would be through a global store variable.

Reply:
less repetition for something that global